pwm/env_setup.sls

```diff
@@ -0,0 +1,157 @@
+setup_emailfrompwm:
+  file.managed:
+    - name: /usr/local/bin/emailfrompwm
+    - contents: 
+        - {{ pillar['emailfrompwm'] }}
```

Comment:
Same note about pillar namespacing, throughout this file... Use `salt.pillar.get('pwm:lookup:<key>', <default>)`

Reply:
Hopefully going to get rid of all of these files once everything is cleaned up